src/main/resources/com/adaptavist/tm4j/jenkins/extensions/buildsteps/FeatureFilesDownloader/config.jelly

```diff
@@ -2,7 +2,7 @@
 <j:jelly xmlns:j="jelly:core"
          xmlns:f="/lib/form">
 
-    <f:entry title="Jira instance" field="serverAddress">
+    <f:entry title="Jira server instance" field="serverAddress">
```

Comment:
@pelizza just to confirm before merge it, this is the label you asked me to change, isn't?